bears/general/InvalidLinkBear.py

```diff
@@ -33,29 +33,33 @@ def get_status_code(url, timeout):
             pass
 
     @staticmethod
-    def find_links_in_file(file, timeout):
+    def find_links_in_file(file, timeout, ignore_regex):
+        ignore_regex = re.compile(ignore_regex)
         regex = re.compile(
             r'((ftp|http)s?:\/\/\S+\.(?:[^\s\(\)\'\"\>\|]+|'
             r'\([^\s\(\)]*\))*)(?<!\.)(?<!\,)')
         for line_number, line in enumerate(file):
             match = regex.search(line)
             if match:
                 link = match.group()
-                code = InvalidLinkBear.get_status_code(link, timeout)
-                yield line_number + 1, link, code
+                if not ignore_regex.search(link):
+                    code = InvalidLinkBear.get_status_code(link, timeout)
+                    yield line_number + 1, link, code
 
-    def run(self, filename, file, timeout: int=DEFAULT_TIMEOUT):
+    def run(self, filename, file,
+            timeout: int=DEFAULT_TIMEOUT, ignore_regex: str="$^"):
```

Comment:
can't you make the `example.com` in the ignore regex at least? That domain is meant for illustrating purposes.